src/controllers/queryRunner.ts

```diff
@@ -736,6 +717,148 @@ export default class QueryRunner {
             );
         }
 
+        await this.writeStringToClipboard(copyString);
+    }
+
+    public async sendToClipboard(
```

Comment:
This function appears to be doing much more than `sendToClipboard`, can we get a better name for this?